src/app/public/src/modules/page-anchor/page-anchor.service.ts

```diff
@@ -7,21 +7,20 @@ import { StacheWindowRef } from '../shared';
 export class StachePageAnchorService implements OnDestroy {
   public anchorStream$ = new Subject <StacheNavLink[]>();
   public pageAnchors: BehaviorSubject<StacheNavLink>[] = [];
-  public bodyHeightChanged = new BehaviorSubject<number>(0);
+  public refreshRequested = new Subject();
```

Comment:
Changing public methods and properties on a service is technically a breaking change. Would it still make sense to retain the name `bodyHeightChanged`?

Reply:
This is a change to the unreleased branch so it will not be a breaking change.